go-controller/pkg/cluster/gateway_shared_intf.go

```diff
@@ -343,5 +344,36 @@ func cleanupSharedGateway() error {
 			return fmt.Errorf("Failed to delete port %s stderr:%s (%v)", port, stderr, err)
 		}
 	}
+
+	// Get the OVS bridge name from ovn-bridge-mappings
+	stdout, stderr, err = util.RunOVSVsctl("--if-exists", "get", "Open_vSwitch", ".",
+		"external_ids:ovn-bridge-mappings")
+	if err != nil {
+		return fmt.Errorf("Failed to get ovn-bridge-mappings stderr:%s (%v)", stderr, err)
+	}
+	// skip the existing mapping setting for the specified physicalNetworkName
+	bridgeName := ""
+	bridgeMappings := strings.Split(stdout, ",")
+	for _, bridgeMapping := range bridgeMappings {
+		m := strings.Split(bridgeMapping, ":")
+		if network := m[0]; network == util.PhysicalNetworkName {
+			bridgeName = m[1]
+			break
+		}
+	}
+	if len(bridgeName) == 0 {
+		return nil
+	}
+
+	// replace all the default connection tracking flows and NodePort related flows and
+	// just add a NORMAL action flow
+	replaceFlowFile := "/tmp/ovnkube_replace_flows.txt"
```

Comment:
can't we just use `del-flows bridgeName`?

Reply:
We still need the NORMAL action flow.

Reply:
Let's try cmd.SetStdin(<bytes.Buffer of the input>)